### PR TITLE
Translated introduction to T&Cs, security page and restricted activities

### PR DESCRIPTION
**Pull request by jutta-frieden**
Translated the introduction sentences for Ts&Cs for German website (Ts&Cs remain English for now).
Unfortunately, the new file does not show properly on the website but gives me the following error code in my browser. Help please. Thank you!

Error: Invariant Violation: Cannot find a route named "security"
   at invariant (/Users/Jutta/splash-pages/node_modules/react/lib/invariant.js:42:15)
   at Function.makePath (/Users/Jutta/splash-pages/node_modules/react-router/lib/createRouter.js:199:11)
   at Function.makeHref (/Users/Jutta/splash-pages/node_modules/react-router/lib/createRouter.js:212:27)
   at Link.getHref (/Users/Jutta/splash-pages/node_modules/react-router/lib/components/Link.js:74:34)
   at Link.render (/Users/Jutta/splash-pages/node_modules/react-router/lib/components/Link.js:99:20)
   at [object Object].ReactCompositeComponentMixin._renderValidatedComponentWithoutOwnerOrContext (/Users/Jutta/splash-pages/node_modules/react/lib/ReactCompositeComponent.js:789:34)
   at [object Object].ReactCompositeComponentMixin._renderValidatedComponent (/Users/Jutta/splash-pages/node_modules/react/lib/ReactCompositeComponent.js:816:14)
   at [object Object].wrapper [as _renderValidatedComponent] (/Users/Jutta/splash-pages/node_modules/react/lib/ReactPerf.js:70:21)
   at [object Object].ReactCompositeComponentMixin.mountComponent (/Users/Jutta/splash-pages/node_modules/react/lib/ReactCompositeComponent.js:237:30)
   at [object Object].wrapper [as mountComponent] (/Users/Jutta/splash-pages/node_modules/react/lib/ReactPerf.js:70:21)

### CODE REVIEW

**Comment by jutta-frieden** [comment]
@octaveauger @isaacseymour do you think one of you could take a look at this and help me resolve the blocker? Cheers

**Comment by octaveauger** [comment]
@jutta-frieden you link to the security page in your new T&Cs page, but have no route for it. You should add a German route in:
`[Security, { name: 'security' }, {
      en: {
        path: '/security',
      },
      fr: {
        path: '/securite',
      },
    },
  ],`

And translate that page (or at the very least create a DE version in English.

**Comment by octaveauger** [comment]
@jutta-frieden I've added a placeholder version which should no longer crash your legal page. Translate the route and page in German.

**Comment by jutta-frieden** [comment]
@octaveauger Great thanks. I had tried this on Saturday and could not get to the final working version. Managed this now with your version! Thank you :-)
PS: only downside is I'll have to translate security page and restricted activities now too :P

**Comment by jutta-frieden** [comment]
@georg911 Hi Georg, could you have a quick look through the copy on /sicherheit, as well as /legal/agb 3 paragraphs at the top? Thank you!
@isaacseymour happy to look through the code for me? I have a feeling there are still some edge-case tests failing...
Thanks guys!

**Comment by georg911** [comment]
@jutta-frieden looking good!

**Comment by isaacseymour** [comment]
@samjewell could you take a look? I haven't worked on splash pages before

File: app/pages/legal/merchants/legal-merchants.de.js [hunk 1]
```diff
@@ -0,0 +1,699 @@
...
+          responsibility to update us with any new contact information, including any
+          change in address. We will send any correspondence to the most recent email
+          address or postal address that you have provided to us for your account and,
+          therefore, you must advise us promptly of any change of your contact details
+          in the interests of security (including name or address) and provide appropriate
+          supporting evidence as required by us.
+        </p>
+      </Translation>
+    );
+  }
```
Comment (jackfranklin) [inline app/pages/legal/merchants/legal-merchants.de.js]: is it right that this file is half German and half English?

File: app/pages/legal/merchants/legal-merchants.de.js [hunk 2]
```diff
@@ -0,0 +1,699 @@
...
+          You are prohibited from doing any of the following:
+        </p>
+        <ul className='list'>
+          <li>
+            selling, transferring or otherwise allowing third parties to access
+            your GoCardless Account except as set out under 'Permitted third party
+            uses of your account'; and
+          </li>
+          <li>
+            collecting any customer's personal data that is accessed through
```
Comment (samjewell) [inline app/pages/legal/merchants/legal-merchants.de.js]: Would you consider using curved apostrophes in the copy rather than straight ones? ‘ and ’ rather than ' ?
It should make for better looking copy on the pages themselves, as well as making the code easier to update later (doing find > replace etc). Here and on subsequent lines below.

File: app/pages/legal/merchants/legal-merchants.de.js (hunk 1 above)
Comment (jutta-frieden) [inline app/pages/legal/merchants/legal-merchants.de.js]: @jackfranklin Yes, that's correct. Same for France. We would love to get the terms full translated but would need legal help. This is the bootstrapped version 1 to have at least SOME terms live.

**Comment by samjewell** [comment]
I'm fairly sure the URL is wrong? 
`...gc.com/de-de/legal/datenschutz/` should probably be `...gc.com/de-de/rechtliche/datenschutz/`

**Comment by samjewell** [comment]
Is the footer right? Does Impressum mean introduction in this case?
screen shot 2015-09-02 at 15 51 13
I was expecting this also to say `rechtliche` rather than `impressum`

File: app/pages/legal/merchants/legal-merchants.de.js (hunk 1 above)
Comment (jackfranklin) [inline app/pages/legal/merchants/legal-merchants.de.js]: Great, thanks for clarifying! :+1:

File: app/messages/de.js [hunk 3]
```diff
@@ -216,9 +216,31 @@ export default {
...
+    description: 'Legal',
+  },
   legal_privacy: {
     title: 'Datenschutzerklärung',
     nav_title: 'Datenschutzerklärung',
+    footer_nav_title: 'Datenschutzerklärung',
     description: 'Datenschutzerklärung',
   },
+  legal_restrictions: {
+    title: 'Our terms of service - Restrictions',
```
Comment (samjewell) [inline app/messages/de.js]: I was expecting this to be in German, no?

Comment (jutta-frieden) [inline app/messages/de.js]: Changed this, thanks Sam!

**Comment by samjewell** [comment]
I can't comment on the line, as it isn't one you've changed in this PR, but line 213:
`title: 'Rechtliche Hinweise',`
I feel like this should now be `Impressum` following the format you've used for the other pages

File: app/messages/de.js (hunk 3 above)
Comment (jutta-frieden) [inline app/messages/de.js]: Let's push changes to footer to next version. This is more complicated for Germany

File: app/messages/de.js [hunk 4]
```diff
@@ -216,9 +216,31 @@ export default {
     description: 'Rechtliche Hinweise',
     sidebar_text: 'Haben Sie Fragen? Rufen Sie uns an unter',
   },
+  legal_merchants: {
+    title: 'Allgemeine Geschäftsbedingungen (AGB)',
+    nav_title: 'AGB',
+    footer_nav_title: 'AGB',
+    description: 'Legal',
```
Comment (samjewell) [inline app/messages/de.js]: Could rename to `rechtliches` perhaps? Same on the other page descriptions below.
This is for the <meta name="description" content="xx something here xx"> tag right?

Comment (jutta-frieden) [inline app/messages/de.js]: updated to AGB here and respectively in other places, good catch. I assume this is for tag

**Comment by samjewell** [comment]
@jutta-frieden this is awesome, great work! This is :+1: from me with those comments. Lets try to get to the footer and URL bar soon. :cake:
